examples/resources/integration_flyio.tf

```diff
@@ -0,0 +1,13 @@
+resource "doppler_integration_flyio" "prod" {
+  name    = "TF Fly.io"
+  api_key = "fo1_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+}
+
+resource "doppler_secrets_sync_flyio" "backend_prod" {
+  integration = doppler_integration_flyio.prod.id
+  project     = "backend"
+  config      = "prd"
+
+  app_id           = "my-app"
+  restart_machines = true
```

Comment:
Maybe it'd be worth including an example showing `delete_behavior` usage?

Reply:
I thought about this as well but I didn't want to create a ton of noise (as it would need to go in every sync type). It's just one extra line though and it could be useful